### PR TITLE
feat: add changeOnHold

### PR DESCRIPTION
**Pull request by WoodNeck**
## Details
This adds a new option "changeOnHold", which triggers `changed` event on mouse/touch hold while animating.

### CODE REVIEW

File: docs/src/pages/Options.mdx [hunk 1]
```diff
@@ -1050,6 +1050,45 @@ Automatically call **[disableInput()](docs/api/Flicking#disableInput)** on initi
   </ColumnItem>
 </Columns>
 
+### changeOnHold
+Change active panel index on mouse/touch hold while animating.
```
Comment (daybrush) [inline docs/src/pages/Options.mdx:1054]: Is it only necessary for hold? It seems like a good feature to change the activePanel even when not on hold.

**Review by daybrush — APPROVED** [review]
LGTM

**Review by malangfox — APPROVED** [review]
LGTM

File: docs/src/pages/Options.mdx (hunk 1 above)
Comment (WoodNeck) [reply docs/src/pages/Options.mdx:1054]: Umm... like when? Can you show me some example cases?

**Pull request merged**